client/src/components/RelatedItems/relatedItemsData.js

```diff
@@ -1,3 +1,131 @@
-module.exports = {
+const relatedItemsdata = {
+  currentProductInfo: {
+    "id": 39333,
+    "campus": "hr-nyc",
+    "name": "Camo Onesie",
+    "slogan": "Blend in to your crowd",
+    "description": "The So Fatigues will wake you up and fit you in. This high energy camo will have you blending in to even the wildest surroundings.",
+    "category": "Jackets",
+    "default_price": "140.00",
+    "created_at": "2021-08-13T14:38:24.136Z",
+    "updated_at": "2021-08-13T14:38:24.136Z",
+    "features": [
+        {
+            "feature": "Fabric",
+            "value": "Canvas"
+        },
+        {
+            "feature": "Buttons",
+            "value": "Brass"
+        }
+    ]
+  },
+  relatedProducts: [
+    39334,
+    39335,
+    39340,
+    39339
+  ],
+  relatedProductsInfo: [
+    {
+      "id": 39334,
+      "campus": "hr-nyc",
+      "name": "Bright Future Sunglasses",
+      "slogan": "You've got to wear shades",
+      "description": "Where you're going you might not need roads, but you definitely need some shades. Give those baby blues a rest and let the future shine bright on these timeless lenses.",
+      "category": "Accessories",
+      "default_price": "69.00",
+      "created_at": "2021-08-13T14:38:24.136Z",
+      "updated_at": "2021-08-13T14:38:24.136Z",
+      "features": [
```

Comment:
Thanks for the suggestion, Aaron.
I skipped it because it didn't seem very clear to me by reading the documentation what properties I should include in the ComparisonModal component. I was thinking about adding only the product features, but it seems like I need to add other characteristics based on the product overview.
It says:
_"The characteristics to be compared are the same as those which appear on the Overview module for each product separately."_
_"These characteristics can either be facts regarding the product or values for which the product has a specific quantity."_